apps/minaintyg/src/schema/error.schema.ts

```diff
@@ -2,3 +2,24 @@ import { z } from 'zod'
 
 export const ErrorType = z.enum(['unknown', 'login-failed', 'unavailable', 'logged-out', 'not-found'])
 export type ErrorTypeEnum = z.infer<typeof ErrorType>
+
+export const ErrorCode = z.enum([
+  'CLIENT_ERROR',
+  'LOGIN_FAILED',
+  'UNKNOWN_INTERNAL_ERROR',
+  'FETCH_ERROR',
```

Comment:
Could ```FETCH_ERROR``` instead be called ```SERVICE_UNAVAILABLE_ERROR```?

Reply:
FETCH_ERROR is from redux, could remap them but would rather not if not necessary 
![image](https://github.com/sklintyg/frontend/assets/28404/3ae6ec8c-0a00-4837-92e1-c5bc0da79f00)